e2e/app/page/admin-workspace-page.ts

```diff
@@ -72,23 +82,6 @@ export default class WorkspaceAdminPage extends AuthenticatedPage {
     return cloudStorageColNames;
   }
 
-  async getAllHeadings2(): Promise<string[]> {
```

Comment:
Deleted these function since they were only verifying the Headings of the workspace-admin page.